lib/init/config-initializer.js

```diff
@@ -445,114 +450,121 @@ function askInstallModules(modules, packageJsonExists) {
     });
 }
 
-/* istanbul ignore next: no need to test inquirer */
+/* istanbul ignore next: no need to test enquirer */
 /**
  * Ask use a few questions on command prompt
  * @returns {Promise} The promise with the result of the prompt
  */
 function promptUser() {
 
-    return inquirer.prompt([
+    return enquirer.prompt([
         {
-            type: "list",
+            type: "select",
             name: "purpose",
             message: "How would you like to use ESLint?",
-            default: "problems",
+            initial: 1,
             choices: [
-                { name: "To check syntax only", value: "syntax" },
-                { name: "To check syntax and find problems", value: "problems" },
-                { name: "To check syntax, find problems, and enforce code style", value: "style" }
+                { message: "To check syntax only", name: "syntax" },
+                { message: "To check syntax and find problems", name: "problems" },
+                { message: "To check syntax, find problems, and enforce code style", name: "style" }
             ]
         },
         {
-            type: "list",
+            type: "select",
             name: "moduleType",
             message: "What type of modules does your project use?",
-            default: "esm",
+            initial: 0,
             choices: [
-                { name: "JavaScript modules (import/export)", value: "esm" },
-                { name: "CommonJS (require/exports)", value: "commonjs" },
-                { name: "None of these", value: "none" }
+                { message: "JavaScript modules (import/export)", name: "esm" },
+                { message: "CommonJS (require/exports)", name: "commonjs" },
+                { message: "None of these", name: "none" }
             ]
         },
         {
-            type: "list",
+            type: "select",
             name: "framework",
             message: "Which framework does your project use?",
-            default: "react",
+            initial: 0,
             choices: [
-                { name: "React", value: "react" },
-                { name: "Vue.js", value: "vue" },
-                { name: "None of these", value: "none" }
+                { message: "React", name: "react" },
+                { message: "Vue.js", name: "vue" },
+                { message: "None of these", name: "none" }
             ]
         },
         {
-            type: "confirm",
+            type: "toggle",
             name: "typescript",
             message: "Does your project use TypeScript?",
-            default: false
+            enabled: "Yes",
+            disabled: "No",
+            initial: 0
         },
         {
-            type: "checkbox",
+            type: "multiselect",
             name: "env",
-            message: "Where does your code run?",
-            default: ["browser"],
+            message: "Where does your code run? (Press <space> to select, <a> to toggle all, <i> to invert selection)",
```

Comment:
@Siilwyn 
Oh, I found that `enquirer` support `hint` option. It's non-blocking suggestion :) 

```suggestion
            message: "Where does your code run?",
            hint: "(Press <space> to select, <a> to toggle all, <i> to invert selection)",
```
* message
<img width="704" alt="스크린샷 2020-05-06 오전 1 40 04" src="https://user-images.githubusercontent.com/41323220/81092583-c473fe80-8f3b-11ea-8055-1d90136929d6.png">

* hint
<img width="707" alt="스크린샷 2020-05-06 오전 1 40 51" src="https://user-images.githubusercontent.com/41323220/81092615-d2c21a80-8f3b-11ea-8b59-e11ed1058b55.png">

Reply:
Nice! I missed that bit.